azext_iot/iothub/params.py

```diff
@@ -246,3 +247,11 @@ def load_iothub_arguments(self, _):
             options_list=["--content-type", "--ct"],
             help="MIME Type of file.",
         )
+
+    with self.argument_context("iot hub certificate root-authority set") as context:
```

Comment:
The -n parameter for iothub_name needs to be added

Reply:
it is inherited from "iot" - https://github.com/Azure/azure-iot-cli-extension/blob/73f5ff3cb283bb8106db2dd81b126b8d25c24b1a/azext_iot/_params.py#L95

So this shouldn't be breaking the pylint - otherwise other commands within the iothub folder that use hub name would also trigger this.

Reply:
Adding this in - there may be scope mismatch since this command may inherit from iot hub certificate